index.html

```diff
@@ -0,0 +1,130 @@
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="hero">
+      <h3 class="hero-text-1">Greetings! welcome to</h3>
+      <h1 class="hero-header-1 mt-20">KRAMERICA INDUSTRIES</h1>
+      <p class="hero-text-2 mt-20">
+        It's business as usual here at Kramerica Industries, we are a space
+        value creation company that provides people with inspiring and exciting
+        experiences by fusing digital technology and real space. Lorem ipsum,
+        dolor sit amet consectetur adipisicing elit. Totam explicabo libero
+        neque cupiditate est dolore earum repellat illo voluptatem rerum ea, ab
+        suscipit animi, voluptates incidunt! Laboriosam nulla vel nostrum?
+      </p>
+      <p class="hero-text-3 mt-20">
+        Launching Our Oil Bladder System <br />
+        @ Burj Khalifa
+      </p>
+      <h2 class="hero-header-2 mt-20">2022.03.30(THUR) ~ 09.16</h2>
+    </main>
+
+    <section class="services mt-20">
+      <h2 class="services-heading">Our Services</h2>
+      <ul id="services-ul"></ul>
+      <a href="#" class="services-button mt-20">Need our products?</a>
+    </section>
+
+    <section class="employees mt-20">
+      <h2 class="employees-heading">Our Employees</h2>
+      <ul id="employees-ul" class="mt-20"></ul>
+      <button type="button" class="employees-button mt-20">
+        More <img src="assets/icons/more.svg" alt="load more employees" />
+      </button>
+    </section>
+
+    <section class="index-partners mt-20">
+      <h2 class="index-partners-heading">Partner</h2>
+      <ul class="mt-20">
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-41.svg" alt="vandelay" />
+        </li>
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-6.svg" alt="bluth.png" />
+        </li>
+        <li class="index-partners-item">
+          <img
+            src="assets/icons/logoipsum-logo-25.svg"
+            alt="dundermifflin.jpg"
+          />
+        </li>
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-8.svg" alt="fightmilk.png" />
+        </li>
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-53.svg" alt="lospolos.png" />
+        </li>
+      </ul>
+    </section>
+
+    <footer class="index-footer mt-20">
+      <div class="index-footer-1">
+        <img src="assets/icons/copyright.svg" alt="copyright" />
+        <h2>KRAMERICA INDUSTRIES</h2>
+      </div>
+      <div class="index-footer-2">
+        <p>2022 Kramerica Industries, All Rights Reserved</p>
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Veritatis
+          fuga iure debitis commodi iste, nobis repellat minus perferendis modi
+          voluptatum! Deleniti corrupti inventore temporibus quaerat laboriosam
+          perspiciatis reprehenderit pariatur explicabo.
+        </p>
+      </div>
+    </footer>
```

Comment:
- [ ] As mentioned earlier, please try to stick to the design as much as possible. Here, your footer has too much text which makes your styles break, so please try to remove some text to avoid this issue, and don't forget to properly align the items with the correspondent space in between.